ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -269,6 +270,10 @@ function handle_occupant_access(event, event_type)
         end
         payload.participantJid = occupant.bare_jid;
         payload.participantId = nick_resource;
+        if (is_lobby) then
+            -- manually extract participantId from jid, because the nick resource is different for lobby
+            payload.participantId = occupant.jid:match("^(.-)-");
```

Comment:
This will not work for jigasi.

Reply:
@damencho Hmm, I had the impression jigasi skips the lobby. What won't work? I logged it and it seemed ok, the jid was the same format

```
    "participantJid": "606e2caa-973c-46b3-9910-372f5e8e3723@stage.8x8.vc",
    "nick": "12094370085 (+12094370085)",
```

Reply:
Actually, we don't even emit lobby webhooks for jigasi